src/kagglehub/gcs_upload.py

```diff
@@ -25,6 +22,19 @@
 REQUEST_TIMEOUT = 600
 
 
+class UploadFileInfo:
+    token: str
+
+
+class Directory:
```

Comment:
nit, rename to `UploadDirectoryInfo` to match the be consistent with the `UploadFileInfo` name.